services/test-validators.js

```diff
@@ -96,6 +96,7 @@ const isBuildStatus = Joi.equal(
   'failing',
   'no tests',
   'not built',
+  'never built',
```

Comment:
This message is returned from vso.

From production:

https://img.shields.io/vso/build/larsbrinkhoff/953a34b9-5966-4923-a48a-c41874cfb5f5/1.svg

![](https://img.shields.io/vso/build/larsbrinkhoff/953a34b9-5966-4923-a48a-c41874cfb5f5/1.svg)